pom.xml

```diff
@@ -60,8 +61,7 @@
     <picocli.version>4.5.2</picocli.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <prometheus.simpleclient.version>0.9.0</prometheus.simpleclient.version>
-    <protobuf.version>3.12.0</protobuf.version>
-    <protoc.version>3.12.0</protoc.version>
+    <protobuf.version>3.19.2</protobuf.version>
```

Comment:
Seems native M1 support was introduced in [v3.20.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.1)

Reply:
Not exactly, it is from [v3.17.3](https://github.com/protocolbuffers/protobuf/releases/tag/v3.17.3)

Reply:
3.17.3 is still Rosetta, there is only [protoc-3.17.3-osx-x86_64.zip](https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protoc-3.17.3-osx-x86_64.zip).

In 3.20.1, there is [protoc-3.20.1-osx-aarch_64.zip](https://github.com/protocolbuffers/protobuf/releases/download/v3.20.1/protoc-3.20.1-osx-aarch_64.zip).

Reply:
You should check the [maven repo](https://repo1.maven.org/maven2/com/google/protobuf/protoc/3.17.3/) instead of GitHub release

Reply:
Thanks for the link, seems they both requires Rosetta 2.

```console
$ file protoc-3.17.3-osx-aarch_64.exe protoc-3.20.1-osx-aarch_64.exe 
protoc-3.17.3-osx-aarch_64.exe: Mach-O 64-bit executable x86_64
protoc-3.20.1-osx-aarch_64.exe: Mach-O 64-bit executable x86_64
```

Reply:
it's a funny thing, they just copy and rename the file... both file has seem md5

Reply:
The latest version is right
```
➜  apache-uniffle git:(master) file ~/Downloads/protoc-3.21.2-osx-aarch_64.exe
/Users/chengpan/Downloads/protoc-3.21.2-osx-aarch_64.exe: Mach-O 64-bit executable arm64
```